openmp/runtime/test/tasking/omp_record_replay_deps_multi_succ.cpp

```diff
@@ -0,0 +1,60 @@
+// REQUIRES: ompx_taskgraph
+// RUN: %libomp-cxx-compile-and-run
+#include <omp.h>
+#include <cassert>
+#include <vector>
+
+#define TASKS_SIZE 12
+
+typedef struct ident {
+  void *dummy;
+} ident_t;
+
+#ifdef __cplusplus
```

Comment:
No need to use `#ifdef __cplusplus` since it is already `%libomp-cxx-compile-and-run`